Gemfile

```diff
@@ -10,11 +10,15 @@ group :development do
   gem 'mutant-rspec',  '~> 0.8.8'
   gem 'rake',          '~> 11.1'
   gem 'rspec',         '~> 3.0'
-  gem 'rubocop',       '~> 0.42.0'
   gem 'simplecov',     '~> 0.12.0'
   gem 'yard',          '~> 0.9.5'
   gem 'activesupport', '~> 4.2'
 
+  if RUBY_VERSION >= '2.2'
```

Comment:
And shouldn't we update our travis configuration as well? I mean, the way it is set up below it will still output something on STDOUT which I find confusing.